pkg/api-server/resource_api_client_test.go

```diff
@@ -91,16 +93,16 @@ func putSampleResourceIntoStore(resourceStore store.ResourceStore, name string,
 	Expect(err).NotTo(HaveOccurred())
 }
 
-func createTestApiServer(store store.ResourceStore, config *config_api_server.ApiServerConfig) *api_server.ApiServer {
+func createTestApiServer(store store.ResourceStore, config *config_api_server.ApiServerConfig, defs ...definitions.ResourceWsDefinition) *api_server.ApiServer {
 	// we have to manually search for port and put it into config. There is no way to retrieve port of running
 	// http.Server and we need it later for the client
 	port, err := test.GetFreePort()
 	Expect(err).NotTo(HaveOccurred())
 	config.Port = port
-	defs := []definitions.ResourceWsDefinition{
+	defs = append([]definitions.ResourceWsDefinition{
 		TrafficRouteWsDefinition,
 		definitions.MeshWsDefinition,
-	}
+	}, defs...)
```

Comment:
I'd consider just adding all the definitions so we've got this as close as possible to real server used in the app.

Reply:
done